src/ReactiveDomain.Testing/EventStore/MockStreamStoreConnection.cs

```diff
@@ -173,9 +182,13 @@ private List<RecordedEvent> GetOrCreateStream(string streamName)
                                     long count,
                                     UserCredentials credentials = null)
         {
+            if (!_connected) throw new Exception("Not Connected");
```

Comment:
Let's make this and the other "Not Connected" ones throw an `InvalidOperationException` since that's slightly more specific than a generic `Exception`.

Reply:
sounds good